lib/pbench/server/api/resources/users_api.py

```diff
@@ -69,17 +70,20 @@ def post(self):
 
         # generate email and send to the user
         # Ideally the message should be a template
-        # rendered using render_template
-        msg = Message(subject="Reset your password!",
-                      sender=self.config.get("pbench-server", "email_username"),
-                      recipients=[email],
-                      body=("Please reset your password at http://localhost:8000/reset_password/" + token)
-                      )
-
-        # This function needs to be asynchronous
-        self.mail.send(msg)
+        # rendered using render_template and the hostname
+        # needs to be a variable that points opens up
+        # pbench-dashboard
+        msg = Message(
+            subject="Reset your password!",
+            sender=self.config.get("pbench-server", "mailfrom"),
+            recipients=[email],
+            body=(
+                "Please reset your password at http://localhost:8000/reset_password/"
+                + token
+            ),
```

Comment:
Right, this will need to leverage the header processing I built into `endpoint_configure.py` to point back at the desired hostname and port (based on either direct or reverse-proxy access). And I suspect it should be `/dashboard/#reset_password` or something like that so that the browser will load the dashboard `index.html` and bootstrap the UMI minified Javascript that will route it to your reset page?

Reply:
Yes...